<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfPlatformHistoryPrefWidget</class>
 <widget class="QWidget" name="WkfPlatformHistoryPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>平台历史</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="tracelinesGroupBox_old">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Trace lines allow you to visualize where a platform has been.&lt;/p&gt;&lt;p&gt;Unlike wing ribbons, trace lines will retroactively show where a platform had been prior to being toggled on.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Trace lines allow you to visualize where a platform has been.&lt;/p&gt;&lt;p&gt;Unlike wing ribbons, trace lines will retroactively show where a platform had been prior to being toggled on.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>轨迹线</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>线宽</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="tracelineWidth">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>长度(秒)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="tracelinesLengthLineEdit">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How far back in (simulation) time that a trace line will display the platform's past location for.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How far back in (simulation) time that a trace line will display the platform's past location for.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>着色依据</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="tracelineColorCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="wingRibbonsGroupBox_old">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wing ribbons are a more advanded form of tracing where a platform has been.&lt;/p&gt;&lt;p&gt;Wing ribbons allow you to visualize both where and in what orientation a platform was in, along with a variety of trackable states that the platform was experiencing/performing based on the color of the ribbon.&lt;/p&gt;&lt;p&gt;Unlike trace lines, due to mild performance concerns, wing ribbons will not retroactively show platform states prior to being toggled on.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wing ribbons are a more advanded form of tracing where a platform has been.&lt;/p&gt;&lt;p&gt;Wing ribbons allow you to visualize both where and in what orientation a platform was in, along with a variety of trackable states that the platform was experiencing/performing based on the color of the ribbon.&lt;/p&gt;&lt;p&gt;Unlike trace lines, due to mild performance concerns, wing ribbons will not retroactively show platform states prior to being were toggled on.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>翼带</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How far back in (simulation) time that a wing ribbon will display the platform's past history for.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How far back in (simulation) time that a wing ribbon will display the platform's past history for.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>长度 (秒)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="wingRibbonLengthLineEdit">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How far back in (simulation) time that a wing ribbon will display the platform's past history for.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How far back in (simulation) time that a wing ribbon will display the platform's past history for.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_21">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The scale of the wingribbon width, relative to the planes wingspan. &lt;/p&gt;&lt;p&gt;A value of 1.0 will align the ribbon with the wing tips.&lt;/p&gt;&lt;p&gt;A value of 2.0 will make the ribbon twice as wide as the wingspan.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The scale of the wingribbon width, relative to the planes wingspan. &lt;/p&gt;&lt;p&gt;A value of 1.0 will align the ribbon with the wing tips.&lt;/p&gt;&lt;p&gt;A value of 2.0 will make the ribbon twice as wide as the wingspan.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>宽度比例</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="wingRibbonWidthScaleField">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The scale of the wingribbon width, relative to the planes wingspan. &lt;/p&gt;&lt;p&gt;A value of 1.0 will align the ribbon with the wing tips.&lt;/p&gt;&lt;p&gt;A value of 2.0 will make the ribbon twice as wide as the wingspan.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The scale of the wingribbon width, relative to the planes wingspan. &lt;/p&gt;&lt;p&gt;A value of 1.0 will align the ribbon with the wing tips.&lt;/p&gt;&lt;p&gt;A value of 2.0 will make the ribbon twice as wide as the wingspan.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="colorSchemeComboBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="deathTransparencySlider">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The colors of a wing ribbon may be more transparent on states where the platform is dead.&lt;/p&gt;&lt;p&gt;This only applies to the parts of the wing ribbon created after the platform has died, not the entire wing ribbon.&lt;/p&gt;&lt;p&gt;Max slider (fully right) will be completely invisible, allowing the wing ribbon to no longer appear recorded once a platform dies.&lt;/p&gt;&lt;p&gt;Min slider (fully left) will have no visual change to the ribbon when when the platform dies. Leaving no indication in the wing ribbon that the platform died.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The colors of a wing ribbon may be more transparent on states where the platform is dead.&lt;/p&gt;&lt;p&gt;This only applies to the parts of the wing ribbon created after the platform has died, not the entire wing ribbon.&lt;/p&gt;&lt;p&gt;Max slider (fully right) will be completely invisible, allowing the wing ribbon to no longer appear recorded once a platform dies.&lt;/p&gt;&lt;p&gt;Min slider (fully left) will have no visual change to the ribbon when when the platform dies. Leaving no indication in the wing ribbon that the platform died.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>配色方案</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_22">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The colors of a wing ribbon may be more transparent on states where the platform is dead.&lt;/p&gt;&lt;p&gt;This only applies to the parts of the wing ribbon created after the platform has died, not the entire wing ribbon.&lt;/p&gt;&lt;p&gt;Max slider (fully right) will be completely invisible, allowing the wing ribbon to no longer appear recorded once a platform dies.&lt;/p&gt;&lt;p&gt;Min slider (fully left) will have no visual change to the ribbon when when the platform dies. Leaving no indication in the wing ribbon that the platform died.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The colors of a wing ribbon may be more transparent on states where the platform is dead.&lt;/p&gt;&lt;p&gt;This only applies to the parts of the wing ribbon created after the platform has died, not the entire wing ribbon.&lt;/p&gt;&lt;p&gt;Max slider (fully right) will be completely invisible, allowing the wing ribbon to no longer appear recorded once a platform dies.&lt;/p&gt;&lt;p&gt;Min slider (fully left) will have no visual change to the ribbon when when the platform dies. Leaving no indication in the wing ribbon that the platform died.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>失效时透明度</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtColorButton</class>
   <extends>QPushButton</extends>
   <header>UtQtColorButton.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tracelinesLengthLineEdit</tabstop>
  <tabstop>wingRibbonLengthLineEdit</tabstop>
  <tabstop>wingRibbonWidthScaleField</tabstop>
  <tabstop>colorSchemeComboBox</tabstop>
  <tabstop>deathTransparencySlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
